autoPlay.user.js

```diff
@@ -5,6 +5,7 @@
 // @version 4.3
 // @match *://steamcommunity.com/minigame/towerattack*
 // @match *://steamcommunity.com//minigame/towerattack*
+// @grant GM_xmlhttpRequest
```

Comment:
shit, that was in the initial commit, didn't remove it again

man. 15. jun. 2015 kl. 19.53 skrev Pavel <notifications@github.com>:

> In autoPlay.user.js
> <https://github.com/SteamDatabase/steamSummerMinigame/pull/154#discussion_r32447423>
> :
>
> > @@ -5,6 +5,7 @@
> >  // @version 4.3
> >  // @match *://steamcommunity.com/minigame/towerattack*
> >  // @match *://steamcommunity.com//minigame/towerattack*
> > +// @grant GM_xmlhttpRequest
>
> why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SteamDatabase/steamSummerMinigame/pull/154/files#r32447423>
> .
>